charts/dslorchestrator/values.yaml

```diff
@@ -427,6 +427,16 @@ istio:
       - PUT
       - DELETE
 
+types:
+  - name: full
```

Comment:
add `suspend: false` for full testlevel and
add `suspend: false` for sanity testlevel